src/admin/rightPanel.tsx

```diff
@@ -99,6 +99,7 @@ export class RightPanel extends React.Component<{}, RightPanelState> {
         this.eventManager.addEventListener('onDataChange', this.onDataChange.bind(this));
         window.addEventListener('resize', this.checkScreenSize.bind(this));
         this.router.addRouteChangeListener(() => this.getPageName());
+        document.addEventListener('keydown', this.onModeChange.bind(this));
```

Comment:
Jelena, I think this is overkill, too many events. Let me still add specific event in the core. That would still make sense to have it (and a few other) events that could help hinting the editors.

Reply:
Then I will close this PR and create a new one when the core part is ready.